virttest/env_process.py

```diff
@@ -930,6 +930,8 @@ def preprocess(test, params, env):
     # otherwise throw TestError with respective error message
     try:
         cpu_family = cpu_utils.get_family() if hasattr(cpu_utils, 'get_family') else cpu_utils.get_cpu_arch()
+        if not cpu_family:
+            cpu_family = "unknown"
     except NotImplementedError:
```

Comment:
Hello @sathnaga, depends on https://github.com/avocado-framework/avocado/pull/4239, let's add "FamilyException" here.

Reply:
@sathnaga https://github.com/avocado-framework/avocado/pull/4239 already be merged, could you help update this one accordingly, thanks!

Reply:
@sathnaga https://github.com/avocado-framework/avocado/pull/4239 already be merged, could you help update this one accordingly, thanks!